utils/utils.jsx

```diff
@@ -1895,3 +1897,8 @@ export function adjustSelection(inputBox, e) {
         setSelectionRange(inputBox, selectionStart + 1, selectionEnd - 1);
     }
 }
+
+export function getNextBillingDate() {
+    const nextBillingDate = moment().add(1, 'months').startOf('month');
```

Comment:
If this is always the start of the month, wouldn't that make the date always the 1st?
If so, couldn't we just use the normal `Date` object?
```suggestion
    const nextBillingDate = new Date();
    nextBillingDate.setMonth(nextBillingDate.getMonth() + 1);
    nextBillingDate.setDay(1);
```

I only say that because you generally don't need `moment.js` for most things. They even say that themselves:
https://momentjs.com/docs/#/-project-status/

Reply:
Couple reasons I'd prefer to keep moment as is:
1. Formatting the date is cleaner
2. Moment usually does some magic in the background around localization with its formatting. Things beyond just "translate December to french", but also how the month/day/year is ordered

Reply:
Okay, we can leave as-is for now. Might work on removing `moment.js` throughout the app at some point. But i'm only 1/5 on that.